lib/config-win32.h

```diff
@@ -609,6 +609,12 @@ Vista
 #  endif
 #endif
 
+/* Define if you have the ConnectEx WinSock 2 extension */
+/* This requires a build target of WinXP or newer */
+#if defined(_WIN32_WINNT) && (_WIN32_WINNT >= 0x0501)
```

Comment:
Why do we need this here at all if it's overridden in curl_setup.h anyway?

Reply:
The curl_setup.h logic doesn't define `HAVE_CONNECTEX` though so if we  would remove this section the other one would need polish.